cves/CVE-2013-0861.yml

```diff
@@ -90,14 +100,15 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: d270c3202539e8364c46410e15f7570800e33343
-  note: ''
-- commit: ''
-  note: ''
+  note: "Fix for latest version"
+- commit: 4cd1dad91ae97fe1f0dd534c3f5566787566f137 
+  note: "Fix for version 1.0.4"
 vccs:
 - commit: b437cec143924eb2a7bbcbb7a7ec320d75d2edac
-  note: Identified by archeogit,
-- commit: de6d9b6404bfd1c589799142da5a95428f146edd
-  note: Identified by SZZUnleashed
+  note: |
+    The av_get_channel_layout_nb_channels is being used in coordinance
+    with the channel_layout vector, which is causing memory corruption. 
```

Comment:
Can you elaborate more on what `av_get_channel_layout_nb_channels` does with the `channel_layout` vector that causes memory corruption? In the VCC, it looks like it returns the number of channels for the layout.